include/AudioSoundIo.h

```diff
@@ -70,7 +70,7 @@ class AudioSoundIo : public AudioDevice
 		return QT_TRANSLATE_NOOP( "AudioDeviceSetupWidget", "soundio" );
 	}
 
-	class setupWidget : public AudioDeviceSetupWidget
+	class setupWidget : public gui::AudioDeviceSetupWidget
```

Comment:
Shouldn't this be in the `gui` namespace? This also makes all the `gui::` inside this classe obsolete...

Reply:
This is a nested class inside the class `lmms::AudioSoundIo` and can't be put into a different namespace scope than its container.

Reply:
Un-nest it :grin:

In the end, I'm OK with both, though I think core/gui separation might be more important than nesting here.

Reply:
Aye, unnesting it would make more sense for separation, but I should note that the other audio interface classes (`AudioDummy`, `AudioJack`, `AudioSdl`, etc.) follow the same pattern as this one, with the `setupWidget` nested inside the main class. Let's separate them in a separate PR for proper separation of these separate tasks.